src/elarm_alarmlist.erl

```diff
@@ -151,36 +151,36 @@ al_test_() ->
      ]}.
 
 just_started() ->
-    {ok,State} = init([]),
-    ?assertEqual({{ok,[]}, State}, get_alarms(State)),
-    EvtId = erlang:now(),
-    ?assertEqual({{error, not_active},State}, get_alarm(EvtId, State)),
-    ?assertEqual({{error, not_active},State}, get_alarm(full, disk_1, State)).
+    {ok, State} = init([]),
+    ?assertEqual({{ok, []}, State}, get_alarms(State)),
+    EvtId = erlang:timestamp(),
```

Comment:
Then, @harenson … you will have to somehow patch the meta suite (or maybe this module, by adding `-ignore_xref`and `-dialyzer` attributes) to remove the check for deprecated functions.